src/contracts/TransferManager.js

```diff
@@ -73,4 +76,17 @@ export default class TransferManager extends Contract {
     }
     return result
   }
+
+  async subscribePauseEvents (callback: (values: any) => void) {
+    const callbackInternalPause = (event: Web3Event) => {
+      callback({event: 'pause', values: event.returnValues})
+    }
+    const callbackInternalUnpause = (event: Web3Event) => {
+      callback({event: 'unpause', values: event.returnValues})
+    }
+    return Promise.all([
+      this.subscribe(PAUSE, {}, callbackInternalPause),
+      this.subscribe(UNPAUSE, {}, callbackInternalUnpause)
+    ])
```

Comment:
You are complicating things.
Instead of subscribePauseEvents just define two functions: subscribePause and subscribeUnpause.
And you don't need to use any returnValues for them.
Actually you can even go without any polymath.js updates.